### DISCUSSION TITLE
What is the NuGet package version that support EKS Pod Identity?

### DISCUSSION THREAD

**Discussion by tri-bao** [aws/aws-sdk-net#3214] [Q&A] [answered]
According this this document [https://docs.aws.amazon.com/eks/latest/userguide/pod-id-minimum-sdk.html](https://docs.aws.amazon.com/eks/latest/userguide/pod-id-minimum-sdk.html) , the .NET SDK version 3.7.734.0 supports Pod Identity

However, in the NuGet repo, [https://www.nuget.org/packages/AWSSDK.Core#supportedframeworks-body-tab](https://www.nuget.org/packages/AWSSDK.Core#supportedframeworks-body-tab) the package AWSSDK.Core, at the time of writing, only have version 3.7.302.15

Where can I get the version 3.7.734.0

Thanks,
Bao

**Comment by dscpinheiro** [comment, marked as answer]
Sorry for the confusion, that's the overall SDK version but the number doesn't match the NuGet packages (for reference, it's used when obtaining the artifacts from S3 directly: https://sdk-for-net.amazonwebservices.com/releases/aws-sdk-net8.0-3.7.734.0.zip).

EKS Pod Identities support was added in version `3.7.302` of `AWSSDK.Core`, so any version later than that (such as `3.7.302.15`) should work as well (previous discussion here: https://github.com/aws/aws-sdk-net/discussions/3126#discussioncomment-8256670)
